draft-ietf-tls-ticketrequests.md

```diff
@@ -47,32 +47,33 @@ normative:
-session resumption, especially if such keying material protects early application
-data. Single-use tickets bound the number of parallel connections a client
-may initiate by the number of tickets received from a given server. To address
-this limitation, this document describes a mechanism by which clients may specify
-the desired number of tickets needed for future connections.
+server-side, per-client state. Servers vend an arbitrary number of session tickets
+to clients, at their discretion, upon connection establishment. Clients store and
+use tickets when resuming future connections. This document describes a mechanism by
+which clients may specify the desired number of tickets needed for future connections.
+This extension aims to remove server ambiguity surrounding ticket count while simultaneously
+reducing ticket waste and priming clients for future connection attempts.
 
 --- middle
 
 # Introduction
 
-As per {{RFC5077}}, and as described in {{RFC8446}},
-TLS servers send clients session tickets at their own discretion in NewSessionTicket messages.
-Clients are in complete control of how many tickets they may use when establishing
-future and subsequent connections. For example, clients may open multiple TLS connections to the same server
-for HTTP, or may race TLS connections across different network interfaces.
-The latter is especially useful in transport systems that implement Happy Eyeballs {{RFC8305}}.
-Since connection concurrency and resumption is controlled by clients, a standard mechanism
-to request more than one ticket is desirable.
-
-This document specifies a new TLS extension -- ticket_request -- that may be used
+As per {{RFC5077}}, and as described in {{RFC8446}}, TLS servers send clients an arbitrary
+number of session tickets at their own discretion in NewSessionTicket messages. There are
+two limitations with this design. First, servers typically guess (or hard-code) the number
+of tickets vended per connection. Second, clients do not have a way of expressing their
+desired number of tickets, which may impact future connection establishment.
+For example, clients may open multiple TLS connections to the same server for HTTP,
+or may race TLS connections across different network interfaces. The latter is especially
+useful in transport systems that implement Happy Eyeballs V2 {{RFC8305}}. Since clients control
+connection concurrency and resumption, a standard mechanism for requesting more than one
+ticket is desirable.
+
+This document specifies a new TLS extension -- "ticket_request" -- that may be used
 by clients to express their desired number of session tickets. Servers may use this
 extension as a hint of the number of NewSessionTicket messages to vend.
-This extension is only applicable to TLS 1.3 {{!RFC8446}} and future versions of TLS.
+This extension is only applicable to TLS 1.3 {{!RFC8446}}, DTLS 1.3 {{!I-D.ietf-tls-dtls13}},
+and future versions of thereof.
```

Comment:
Is `of thereof` correct english? I would have written `of both`

Reply:
Nope, typo! Fixed.